### ISSUE TITLE
MD057 missing support for `?raw=true` in relative links, config docs have invalid options provided

### ISSUE THREAD

**Issue by tdschwarz** [rvben/rumdl#198]
The relative link checker throws false positives for media content (even with the default exclude on media) for docs referencing static assets with the `?raw=true` arg for rendering directly in GitHub

```markdown
# MD057 false positive for `?raw=true` arg

![Embed link to raw image](path/to/raw/image.png?raw=true)
```

```bash
3:1: [MD057] Relative link 'path/to/raw/image.png?raw=true' does not exist
```

When using the example config from `rumd/docs/md057.md` the following error is thrown

```toml
[MD057]
skip-media-files = true  # Skip checking media files like images/videos (default: true)
```

```bash
[config warning] Unknown option for rule MD057: skip-media-files
```

**Comment by rvben** [comment]
Hi @tdschwarz, thanks for raising this. We should just support queries in file paths like that.

Regarding the docs, that is outdated and I'll update. I removed `skip-media-files` to simplify. Do you think there's a good case where a user might want to disable checking just the media files?

**Comment by tdschwarz** [comment]
> Do you think there's a good case where a user might want to disable checking just the media files?

@rvben thanks for the quick response! The only case I can think of for relative links would be for media files that are generated locally by automated scripts and excluded from the repo by `.gitignore`, but that seems pretty niche.

**Comment by rvben** [comment]
There were two problems with MD057:

1. **Query parameters in relative links** - MD057 was incorrectly flagging relative links with query parameters (like `?raw=true`) as non-existent, even when the target file existed.
2. **Invalid documentation** - The docs referenced a non-existent config option `skip-media-files` that had been removed from the implementation.

Both issues are now fixed and verified in 0.0.195. 🚀
